src/cljs/main/broadfcui/common/method/config_io.cljs

```diff
@@ -22,6 +22,25 @@
 
 (def clip (partial merge table-style/clip-text))
 
+; ask what other types it could be. Maps?
+(defn get-typed [string-input input-type]
+  (if (or (= input-type "int")
+          (string/includes? input-type "array")
```

Comment:
`string/starts-with?` should work and not match things that it shouldn't (e.g. `Map[String,Array]`).